infrastructure/repository/inventory.py

```diff
@@ -62,11 +62,11 @@ def updateTransactionType(self,id,inventoryTransaction) -> None:
             super().update(inventoryTransaction_orm,inventoryTransaction.dict())
         return
 
-    def listInventoryItem(self) -> List[InventoryItemDTO]:
+    def listInventoryItems(self) -> List[InventoryItemDTO]:
         inventoryItems = self.readAll(InventoryItem)
         return [InventoryItemDTO.from_orm(inventoryItem) for inventoryItem in inventoryItems]
 
-    def listPharmacyItem(self) -> List[PharmacyItemDTO]:
+    def listPharmacyItems(self) -> List[PharmacyItemDTO]:
```

Comment:
The changes made here have to be changed in core/protocol/inventory.py. Inventory protocol is like an interface of inventory repository.